packages/components/src/internal/components/domainproperties/DomainRowExpandedOptions.tsx

```diff
@@ -160,9 +160,10 @@ export class DomainRowExpandedOptions extends React.Component<IDomainRowExpanded
                         defaultScale={field.defaultScale}
                         onChange={onChange}
                         lockType={field.lockType}
-                        scannable={field.scannable}
-                        appPropertiesOnly={appPropertiesOnly}
-                        showScannableOption={domainFormDisplayOptions?.showScannableOption}
+                        ////Issue #44567: Hide scannable option due to matching issues with floating point representation.
```

Comment:
Hmm...I'm torn on if these commented out lines should stay in the code or not. I assume the reason to leave them is incase we ever decide to go back on this. I think just putting the issue number comment here is enough as that should lead to the issue tracker which will have the PR link.

Reply:
I think I'll leave the comment and make the `showScannableOption={false}` explicit.

Reply:
Yup, that sounds good